backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobExecutionActor.{AbortedResponse, BackendJobExecutionResponse, ExecuteJobCommand, FailedNonRetryableResponse}
+import cromwell.backend.BackendLifecycleActor._
+
+import scala.concurrent.Future
+import scala.util.Try
+
+
+trait BackendCacheHitCopyingActor extends Actor with ActorLogging with BackendJobLifecycleActor {
+
+  def copyCachedOutputs: Future[BackendJobExecutionResponse]
+
+  def receive: Receive = LoggingReceive {
+    case ExecuteJobCommand =>
+      val shutdown: PartialFunction[Try[BackendJobExecutionResponse], Unit] = { case _ => context stop self }
+      performActionThenRespond(copyCachedOutputs andThen shutdown, onFailure = cachingFailed)
+    case AbortJobCommand =>
+      abort()
+      context.parent ! AbortedResponse(jobDescriptor.key)
+      context.stop(self)
```

Comment:
I was just thinking that i prefer it that way, but the other way fits in better w/ the official style guide.

OTOH I also prefer `foo.map(x => ...)` but have switched to `foo map { x => ... }` due to the official guide as well, so my bias is evident.